src/token/erc2981/erc2981.cairo

```diff
@@ -0,0 +1,176 @@
+// SPDX-License-Identifier: MIT
```

Comment:
While the idea is good, we try to be as consistent as possible with modules that we have already implemented in Solidity, and we have an implementation [here](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/05f218fb6617932e56bf5388c3b389c3028a7b73/contracts/token/common/ERC2981.sol).

Reply:
Do you mean that I should removed my `FeesRatio` structure and use `_feeDenominator` in storage as in solidity contract ?

Reply:
The solidity implementation doesn't have `_feeDenominator` in storage, but as part of the bytecode as a function. I mean to be as consistent as possible, only diverging when the benefit is obvious or there is a feature inconsistency.

Reply:
Thanks for your explanation, I have updated source code to be more consistent with solidity implementation.